firefox_desktop/firefox_desktop.model.lkml

```diff
@@ -20,6 +21,12 @@ explore: +client_counts {
       client_counts.sample_id: "0"
     ]
   }
+  join: felt_privacy_pinned_retention_2_week {
```

Comment:
I decided that since daniel's query runs on clients daily, and returns client_id-submission_date pairs, that we should just join the results back onto the client counts explore since that will allow further segmentation of the users for this measure. Am open to counter-arguments

Reply:
@danielkberry [here's an example of how this works in creating the chart](https://mozilla.cloud.looker.com/explore/firefox_desktop/client_counts?qid=WNnjoHYorWJrXJCYUdZPeU&toggle=fil) - you have to check this branch out in looker from the git menu to be able to see it